extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-build-items.js

```diff
@@ -52,7 +57,11 @@ export class QwcBuildItems extends LitElement {
   }
 
   render() {
-    return html`${until(this._render(), html`<span>Loading build items...</span>`)}`;
+      if (this._buildStepsMetrics && this._filtered) {
+          return this._render();
+      }else {
+          return html`<span>Loading build items...</span>`;
+      }
```

Comment:
@phillip-kruger quick question: so the render method is called repeatedly by default?

Reply:
Render gets called on startup and when any `state: true` property change